src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -0,0 +1,188 @@
+import { Component } from '../Base/Component';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { Initialization } from '../Base/Initialization';
+import { exportGlobally } from '../../GlobalExports';
+import 'styling/_SimpleFilter';
+import {$$, Dom} from "../../utils/Dom";
+import {
+    IBuildingQueryEventArgs, IDoneBuildingQueryEventArgs, IQuerySuccessEventArgs,
+    QueryEvents
+} from "../../events/QueryEvents";
+import {ComponentOptions, IFieldOption} from "../Base/ComponentOptions";
+import {l} from "../../strings/Strings";
+import {Assert} from "../../misc/Assert";
+import * as _ from 'underscore';
+import {Checkbox} from "../FormWidgets/Checkbox";
+import {IGroupByRequest} from "../../rest/GroupByRequest";
+import {QueryBuilder} from "../Base/QueryBuilder";
+import {IStringMap} from "../../rest/GenericParam";
+import {Utils} from "../../utils/Utils";
+import {BreadcrumbEvents, IClearBreadcrumbEventArgs, IPopulateBreadcrumbEventArgs} from "../../events/BreadcrumbEvents";
+import {BreadcrumbValueElement} from "../Facet/BreadcrumbValueElement";
+import {BreadcrumbValueList} from "../Facet/BreadcrumbValuesList";
+
+
+export interface ISimpleFilterOptions{
+    title: string;
+    values: string[],
+    field: IFieldOption,
+    //includeInBreadcrumb: boolean,
+    valueCaption?: any
+}
+
+export class SimpleFilter extends Component {
+    static ID = 'SimpleFilter';
+    static doExport = () => {
+        exportGlobally({
+            'SimpleFilter': SimpleFilter
+        })
+    };
+    static options: ISimpleFilterOptions = {
+
+        values: ComponentOptions.buildListOption<string>(),
+        field: ComponentOptions.buildFieldOption({ required: true, groupByField: true, section: 'Identification' }),
+        title: ComponentOptions.buildStringOption({defaultValue: l('NoTitle')}),
+        //includeInBreadcrumb: ComponentOptions.buildBooleanOption({defaultValue : true}),
+        valueCaption: ComponentOptions.buildCustomOption<IStringMap<string>>(() => {
+            return null;
+        }),
+    };
+
+    public checkboxes: {checkbox: Checkbox, value: string} [] =  [];
+    public checkboxContainer: Dom;
+    private expanded = false;
+
+    constructor(public element: HTMLElement, public options: ISimpleFilterOptions, public bindings?: IComponentBindings){
+        super(element, SimpleFilter.ID, bindings);
+        this.type = 'SimpleFilter';
+        this.options = ComponentOptions.initComponentOptions(element, SimpleFilter, options);
+
+        let select = $$('select', {className: 'select'});
+        let option = $$('option', {className: 'option'});
+        option.el.innerText = l(this.options.title);
+        select.el.appendChild(option.el) ;
+
+        let overselect = $$('div', {className: 'overselect'});
```

Comment:
never use unprefixed css classes. Always `coveo-`.

Also, try to use a specific class that include your component name for example, so that you can use as much flat css rules as possible.